bots/images/scripts/debian.setup

```diff
@@ -71,6 +71,7 @@ case "$RELEASE" in
         # https://tracker.debian.org/pcp not currently in testing
         COCKPIT_DEPS="${COCKPIT_DEPS/libpcp3 /}"
         COCKPIT_DEPS="${COCKPIT_DEPS/pcp /}"
+        TEST_PACKAGES="${TEST_PACKAGES} libblockdev-mdraid2"
```

Comment:
libblockdev-mdraid2 is a run-time dependency of Cockpit, no?  Without it, the mdraid features of Cockpit are broken, and mdraid isn't optional (yet) in the UI.  Also, mdraid is not a plugin for udisks2, right?

So shouldn't some package depend on liblockdev-mdraid2 as well?  Maybe cockit-storaged, maybe udisks2.

Reply:
udisks2 itself doesn't need the mdraid plugin; the dependency was recently demoted from Recommends: to Suggests:, see http://bugs.debian.org/873706 .

If cockpit-storaged requires it, then it should depend on libblockdev-mdraid2. I'll adjust the dependencies accordingly.

Reply:
Updated, please have a look?